sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -208,6 +210,8 @@ def _add_entity_properties(source):
     properties = {}
 
     to_send = dict(source)  # shallow copy
+    properties["PartitionKey"] = to_send.pop("PartitionKey", None)
+    properties["RowKey"] = to_send.pop("RowKey", None)
```

Comment:
I don't think these should have a fallback of None.
I think this would be more accurate:
```python
try:
    properties["PartitionKey"] = to_send.pop("PartitionKey")
except KeyError:
    pass
try:
    properties["RowKey"] = to_send.pop("RowKey")
except KeyError:
    pass


```

Reply:
I have updated but put them in one `try except` block.